### DISCUSSION TITLE
Plugin / package repository?

### DISCUSSION THREAD

**Discussion by lard** [lem-project/lem#1406] [Q&A]
Hi,

Congratulations on this amazing project!  Just got it running my 10 year old laptop and it's nice and responsive and looks very slick so far.

I was wondering if there was a [MELPA](https://www.melpa.org/)  like repository for community extensions or if there were any plans to encourage such an ecosystem.  Just dreaming of a day when Lem can be to Emacs what Neovim is to Vim (maybe without 100% elisp compatibility ;-) ).

I've always resented having to set brain capacity aside for dealing with elisp which is no use outside of Emacs but CL is a great everyday language to be fluent in.

**Comment by hendursaga** [comment]
At the very least, something like an [awesome list](https://github.com/sindresorhus/awesome) would be nice!
